recipe-server/client/control/sass/control.scss

```diff
@@ -90,7 +90,6 @@
   h3 {
     color: $darkBrown;
     display: inline-block;
-    float: left;
```

Comment:
This was causing the first item in a list of possible filters to be much larger than it should have. Removing it didn't seem to cause any other issues.